music21/figuredBass/realizer.py

```diff
@@ -186,10 +186,7 @@ def addLyricsToBassNote(bassNote, notationString=None):
     n = notation.Notation(notationString)
     if not n.figureStrings:
         return
-    maxLength = 0
-    for fs in n.figureStrings:
-        if len(fs) > maxLength:
-            maxLength = len(fs)
+    maxLength = max([len(fs) for fs in n.figureStrings])
```

Comment:
While here, may as well
```suggestion
    maxLength = max(len(fs) for fs in n.figureStrings)
```

Reply:
Using a generator is slower than a list comprehension and really has no particular benefits in this case: https://stackoverflow.com/questions/47789/generator-expressions-vs-list-comprehensions.

Reply:
Ah, good point! Honestly I just wanted to defend against the possibility of a cosmetic lint message for it in the future, but I guess it's unlikely there will be one, given the reasoning you link out to. Thanks.